modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/background/task/WorkflowMetricsSLAProcessBackgroundTaskHelper.java

```diff
@@ -107,21 +105,12 @@ private String _getBackgroundTaskName(
 	@Reference
 	private BackgroundTaskLocalService _backgroundTaskLocalService;
 
-	@Reference(target = ModuleServiceLifecycle.PORTLETS_INITIALIZED)
-	private ModuleServiceLifecycle _moduleServiceLifecycle;
-
 	@Reference
 	private WorkflowMetricsReindexStatusMessageSender
 		_workflowMetricsReindexStatusMessageSender;
 
 	@Reference
 	private WorkflowMetricsSLADefinitionLocalService
 		_workflowMetricsSLADefinitionLocalService;
-
-	@Reference(
-		target = "(background.task.executor.class.name=com.liferay.portal.workflow.metrics.internal.background.task.WorkflowMetricsSLAProcessBackgroundTaskExecutor)"
-	)
-	private BackgroundTaskExecutor
-		_workflowMetricsSLAProcessBackgroundTaskExecutor;
```

Comment:
Do not remove this one, it's used to ensure `WorkflowMetricsSLAProcessBackgroundTaskExecutor` is activated before the method that call the background process is called